.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,17 +14,25 @@
 
 Closes # .
 
+<!-- The next section is mandatory. If your PR doesn't require testing, please indicate that you are purposefully omitting instructions.  Otherwise, please include detailed instructions on how these changes can be tested (including pre-conditions, configuration, steps to take and expected results). It may help to write your instructions using pseudocode -- as if you're telling a computer how to execute the test.-->
+
+- [ ] This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).
+
+<!-- Begin testing instructions -->
+
 ### How to test the changes in this Pull Request:
 
 1.
 2.
 3.
 
+<!-- End testing instructions -->
+
 ### Other information:
 
 -   [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
 -   [ ] Have you written new tests for your changes, as applicable?
--   [ ] Have you successfully run tests with your changes locally?
+-   [ ] Have you successfully run tests with your changes locally (including end to end tests)?
```

Comment:
As the tests are now required in CI, I wonder is this task needed, or perhaps we can reword it to focus on testing of the changes themselves.

Reply:
There's occasionally differences in results between CI and locally, but hopefully that will be less and less as we improve the test suite.  So I agree with you on this point, let's remove it.